doc/sphinx/install/conda.md

```diff
@@ -69,16 +53,11 @@ from Python. For this example, the environment is named `ct-env`. From the comma
 line (or the Anaconda Prompt on Windows), run:
 
 ```shell
-conda create --name ct-env --channel cantera cantera ipython matplotlib jupyter
+conda create --name ct-env cantera ipython matplotlib jupyter
```

Comment:
This is only sufficient if you're using miniforge or have otherwise configured `conda-forge` to be in the default channel list. I think we either need to suggest using the `--channel conda-forge` option or a different way of configuring the channel.

Reply:
Good call. I'll make that change